exercises/practice/pov/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "pov"
+version = "0.1.0"
+edition = "2024"
+
+# Not all libraries from crates.io are available in Exercism's test runner.
+# The full list of available libraries is here:
+# https://github.com/exercism/rust-test-runner/blob/main/local-registry/Cargo.toml
+[dependencies]
+
+[dev-dependencies]
+pretty_assertions = "1.4.1"
```

```suggestion
pretty_assertions = "*"
```

This will make it less likely for solutions to break if pretty_assertions is updated to 2.0 and the test runner doesn't include 1.x anymore. It can still break if pretty_assertions 2.0 doesn't have a macro `assert_eq!` anymore, but that seems even less likely.

also, nit: add trailing newline